xml/version-diff.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```

Comment:
add a copyright notice

Reply:
Added, with the starting copyright date being the year the stuff was added to xml/Makefile.am in the first place.